### DISCUSSION TITLE
Wrong behaviour for visual mode search

### DISCUSSION THREAD

**Discussion by canelhasmateus** [JetBrains/ideavim#765] [General]
I've recently moved from ideavim 2.2 to the latest version. 
In doing so, I've noticed a change of behaviour during searching inside visual mode. 

Normally ( and also in vim ), searching inside visual mode extends the selection up to the found pattern. 
It seems that in Ideavim 2.4+ this ceased to be the case.

**Comment by AlexPl292** [comment]
Hi, this is a known regression. Please follow this issue: [VIM-3090](https://youtrack.jetbrains.com/issue/VIM-3090)
